stuff.js

```diff
@@ -0,0 +1,36 @@
+// uploads image to AWS s3 and updates user schema with url of profile image
```

Comment:
File name made me laugh, but makes it something that is more clear on what it does

Reply:
Actually, can this whole file be deleted? It looks the same as what's in routes/users

Reply:
Whops! I meant to delete that. I had copied it over when I fixing the branch.